pages/dkp/kommander/2.2/insights/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: layout.pug
+navigationTitle: DKP Insights
+title: DKP Insights
+excerpt: Use DKP Insights to monitor and tune your environment
+menuWeight: 55
+beta: false
+enterprise: false
+techPreview: true
+---
+
+<p class="message--note"><strong>NOTE:</strong> We are offering DKP Insights with this release as a Technical preview only. </p>
```

Comment:
The casing of Technical preview feels off to me? Should this be "Technical Preview"? Or lowercase?